tests/linearizability/client.go

```diff
@@ -27,11 +27,12 @@ import (
 )
 
 type recordingClient struct {
-	client  clientv3.Client
-	history *model.AppendableHistory
+	client   clientv3.Client
+	history  *model.AppendableHistory
+	baseTime time.Time
```

Comment:
Can you add a comment explaining why we have baseTime and why time measurements should  relative to baseTime?